package.json

```diff
@@ -23,6 +23,7 @@
     "@angular/platform-browser-dynamic": "^2.4.0",
     "@angular/router": "^3.4.0",
     "core-js": "^2.4.1",
+    "material-design-lite": "^1.3.0",
```

Comment:
We already have @angular/material.
Is it ok to have both from performance and compatibility side?